### DISCUSSION TITLE
Securly extension screen monitoring

### DISCUSSION THREAD

**Discussion by alany08** [3kh0/ext-remover#1083] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
You can disable access to local files in extension settings, for securly. This prevents securly from grabbing the screen of an opened local file. You can then save an HTML of an iframe, then open the local file. Securly either crashes or fails to capture the screen. I had a very chill teacher that showed me their view, and it shows your last open tab or it shows your device as offline.

**Comment by VFXKLIPS** [comment]
can u explain step by step please
